platforms/core-configuration/file-collections/src/main/java/org/gradle/api/internal/file/FileCollectionInternal.java

```diff
@@ -60,6 +61,14 @@ public interface FileCollectionInternal extends FileCollection, TaskDependencyCo
      */
     TreeFormatter describeContents(TreeFormatter formatter);
 
+    /**
+     * Calculates the execution time value of this file collection. The resulting value is serialized to the configuration cache
```

Comment:
It is fine to mention the configuration cache as a use case, but could we define this method's purpose without relying so prominently on CC?

Reply:
We having some similar statement in [ProviderInternal](https://github.com/gradle/gradle/blob/a3dc91fa697b5de5fd9c2f4ce13e1fbda561fa26/platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/ProviderInternal.java#L151). I was struggling to invent some CC-agnostic doc, but it's hard to do for the thing that was designed solely for work in CC.